core/sr-primitives/src/lib.rs

```diff
@@ -480,13 +480,13 @@ macro_rules! impl_outer_config {
 				)*
 			}
 			#[cfg(any(feature = "std", test))]
-			impl $crate::BuildStorage for $main {
+			impl<'a> $crate::BuildStorage for &'a $main {
```

Comment:
In PR description

> The motivation is benchmarks that need to repeatedly initialize new storage backends. It would be nice to do so without rebuilding the config each time (which involves key generation).

Reply:
Yeah I understand this, but why did you not change the trait to take a reference?